Raven.Abstractions/Smuggler/SmugglerFilesApiBase.cs

```diff
@@ -240,12 +235,19 @@ protected SmugglerFilesApiBase(SmugglerFilesOptions options)
                 Operations.ShowProgress("Got Exception during smuggler export. Exception: {0}. ", e.Message);
                 Operations.ShowProgress("Done with reading files, total: {0}, lastEtag: {1}", totalCount, lastEtag);
 
+                cts.Cancel();
+
                 exceptionHappened = new SmugglerExportException(e.Message, e)
                 {
                     LastEtag = lastEtag,
                 };
             }
 
+            getFilesTask.Wait();
+
+            cts.Dispose();
```

Comment:
I've decided to dispose it here (when both getFilesTask and the exoprt loop are finished).  This way I am using the same CTS in both directions (task <-> export loop)